Link to LMS Control Panel from settings

SteveMicroNova opened the pull request:
### What does this change intend to accomplish?
![image](https://github.com/user-attachments/assets/03faaa32-4ccd-4493-bcf7-a01888d8e8b4)
Change LMS toggle switch so that it doesn't download system config on LMS deactivation, include link to LMS server when LMS Mode is active

### Checklist

* [x] Have you tested your changes and ensured they work?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Does your submission pass linting & tests? You can test on localhost using `./scripts/test`
* [x] If this is a UI change, have you tested it across multiple browser platforms on both desktop and mobile?

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

rtertiaer commented on web/src/pages/Settings/Config/ConfigPanels/LMSMode.jsx:
```diff
@@ -20,12 +21,22 @@ export default function LMSMode() {
+            return (
+                <Button variant="contained" sx={{color: 'black'}} target="_blank" href={`http://${window.location.hostname}:9000`}>
+                    LMS Control Panel
+                </Button>
+            )
+        }
+    }
+
     function LMSModal(){
         if(lmsMode){
             return(
                 <ConfigModal
                     body={"This will reset AmpliPi to factory settings, you will have to either manually reconfigure it or reupload the config that was downloaded when LMS mode was initially toggled."}
-                    onApply={() => {ConfigDownload(); LMSModeHandler();}}
+                    onApply={() => {LMSModeHandler();}}
                     open={modalOpen}
```
not a fan of this change: https://github.com/micro-nova/AmpliPi/issues/976#issuecomment-2411833840 & https://github.com/micro-nova/AmpliPi/pull/805

rtertiaer reviewed (changes requested):
is there any way that button could look a little less haphazardly placed? I don't have any suggestions here - I know we're largely not designers - but I'd love to ensure we at least don't make things look worse. some things that I notice:
* it's a button, but doesn't do a button-like thing - it does a link-like thing
* there are two other types of buttons on the page; this is themed like neither of them
* there's zero margin/padding on this button between it and the divider - feels cramped
* it's not horizontally aligned to anything but the prior content, which feels unfinished

SteveMicroNova commented:
> * it's a button, but doesn't do a button-like thing - it does a link-like thing

I'm happy to make it a link again, though we'll probably have to hash that out a little further given Klayton's dislike of my previous link strategy

> * there are two other types of buttons on the page; this is themed like neither of them

Fixed

> * there's zero margin/padding on this button between it and the divider - feels cramped

This is the case for all of the buttons, but obviated by the `contained` styling; corrected in recent commit

> * it's not horizontally aligned to anything but the prior content, which feels unfinished

I've got no good way of fixing this as there isn't really a gridmodel in place for multi-input sections but I've taken a note from the "Upload Config" section and just made there be some spacing between the inputs


![image](https://github.com/user-attachments/assets/da821010-b07c-465a-89c8-716c1e4a59a9)

klay2000 reviewed (approved):
Lincoln, Steve, and I have decided the button-link thing should be fine as long as it opens in a new tab.
